service/src/main/java/greencity/mapping/HabitTranslationMapper.java

```diff
@@ -29,4 +60,22 @@ protected HabitTranslation convert(HabitTranslationDto habitTranslationDto) {
     public List<HabitTranslation> mapAllToList(List<HabitTranslationDto> dtoList) {
         return dtoList.stream().map(this::convert).collect(Collectors.toList());
     }
+
+    /**
+     * Method that build {@link List} of {@link HabitTranslation} from {@link List}
+     * of {@link HabitTranslationDto} and {@link String} language.
+     *
+     * @param dtoList  {@link List} of {@link HabitTranslationDto}
+     * @param language {@link String}
+     *
+     * @return {@link List} of {@link HabitTranslation}
+     *
+     * @author Chernenko Vitaliy
+     */
+    public List<HabitTranslation> mapAllToList(List<HabitTranslationDto> dtoList, String language) {
+        if (language.equals("ua")) {
```

Comment:
please move `ua` to the constant and compare like this to prevent NPE
`"ua".equals(language)`

Reply:
Good point. I'll remember it.
Fixed.